terraso_backend/apps/project_management/migrations/0001_initial.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 4.1.7 on 2023-03-29 21:19
+
+import uuid
+
+import rules.contrib.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = []
+
+    operations = [
+        migrations.CreateModel(
+            name="Site",
+            fields=[
+                ("deleted_at", models.DateTimeField(db_index=True, editable=False, null=True)),
+                ("deleted_by_cascade", models.BooleanField(default=False, editable=False)),
+                (
+                    "id",
+                    models.UUIDField(
+                        default=uuid.uuid4, editable=False, primary_key=True, serialize=False
+                    ),
+                ),
+                ("created_at", models.DateTimeField(auto_now_add=True)),
+                ("updated_at", models.DateTimeField(auto_now=True)),
+                ("slug", models.SlugField(blank=True, editable=False, max_length=250)),
+                ("name", models.CharField(max_length=200)),
+                ("latitude", models.FloatField()),
+                ("longitude", models.FloatField()),
+            ],
+            options={
+                "ordering": ["created_at"],
```

Comment:
question: does this create a SQL index on "created_at"? i guess this is default behavior from a parent class?

Reply:
> does this create a SQL index on "created_at"?

No idea, would be interesting to check it out.

> i guess this is default behavior from a parent class?

Yes, I think I saw you mention this in Slack, it seems to be the case.